src/main/scala/com/fulcrumgenomics/vcf/filtration/FilterSomaticVcf.scala

```diff
@@ -24,190 +24,181 @@
+      |
+      |* `--a-tailing-distance`  (previously `--end-repair-distance`) allows control over how close to the ends of
+      |                          reads/templates errors can be considered to be candidates for the A-tailing artifact.
+      |                          Higher values decrease the power of the test, so this should be set as low as possible
+      |                          given observed errors.
+      |* `--a-tailing-p-value`   (previously `--end-repair-p-value`) the p-value at or below which a filter should be
+      |                          applied. If no value is supplied only the `INFO` annotation is produced and no `FILTER`
+      |                          is applied.
+      |
+      |### End Repair Fill-in Artifact Filter
+      |
+      |The end repair fill-in artifact filter attempts to measure the probability that a single-nucleotide mismatch is
+      |the product of an error in the template generated during the end repair fill-in step that is common to many
+      |Illumina library preparation protocols, in which single-stranded 3' overhangs are filled in to create a blunt
+      |end. These artifacts originate from single-stranded templates containing damaged bases, often as a consequence
+      |of oxidative damage. These DNA lesions, for example 8-oxoguanine, undergo mismatched pairing, which after PCR
+      |appear as mutations at the ends of reads.
+      |
+      |The filter adds the `INFO` field `ERFAP` to records SNVs. This field records the p-value representing the
+      |probability of the null hypothesis (e.g. that the variant is a true mutation), so lower p-values indicate that
+      |the variant is more likely an end repair fill-in artifact. If a threshold p-value is specified, then the `FILTER`
+      |tag `EndRepairFillInArtifact` will be applied to variants with p-values less than or equal to the threshold.
+      |
+      |Two options are available:
+      |
+      |* `--end-repair-fill-in-distance`  allows control over how close to the ends of reads/templates errors can be
+      |                                   considered to be candidates for the artifact. Higher values decrease the
+      |                                   power of the test, so this should be set as low as possible given observed
+      |                                   errors.
+      |* `--end-repair-fill-in-p-value`   the p-value below which a filter should be applied. If no value is supplied
+      |                                   only the annotation is produced and no filtering is performed.
+      |
+      |## Performance Expectations
+      |
+      |By default `--access-pattern` will be set to `RandomAccess` and the input BAM will be queried using index-based
+      |random access. Random access is mandatory if the input VCF is not coordinate sorted. If random access is
+      |requested and the input VCF is not coordinate sorted, then an exception will be raised on the first
```

Comment:
I'm confused, you say `is mandatory if the input VCF is not coordinate sorted`  but then say an exception is raised "the input VCF is not coordinate sorted".  Those seem at odds with each other.